docs/developer_manual/services/developing_an_assemblyline_service.md

```diff
@@ -111,6 +161,29 @@ In your service directory, you will add the YAML configuration file `service_man
       image: ${REGISTRY}testing/assemblyline-service-sample:latest
       cpu_cores: 1.0
       ram_mb: 1024
+
```

Comment:
Again, since this is not required for every service, it not belong in the base service development.